hedera-node/hedera-app/src/test/java/com/hedera/node/app/workflows/handle/HandleWorkflowTest.java

```diff
@@ -170,6 +203,12 @@ void setup(@Mock InstantSource instantSource) {
                 .when(dispatcher)
                 .dispatchHandle(any());
 
+        final var fakeHederaState = new FakeHederaState();
+        fakeHederaState.addService(RecordCacheService.NAME, Map.of("TransactionRecordQueue", new LinkedList<>()));
+        when(workingStateAccessor.getHederaState()).thenReturn(fakeHederaState);
+
+        hederaRecordCache = spy(new RecordCacheImpl(deduplicationCache, workingStateAccessor, configProvider));
```

Comment:
Not part of this PR, but it makes me nervous that we pass `workingStateAccessor` into this constructor. I wouldn't be surprised if this introduces very subtle bugs.